src/components/button/Button.tsx

```diff
@@ -9,6 +9,7 @@ export type ButtonProps = {
   iconButton?: boolean;
    /** Content of the button*/
   children?: React.ReactNode;
+  /** Optional CSS class name */
```

Comment:
It's quite obvious what a classname is so maybe the comment is a bit unnecessary.

Reply:
Yeah, maybe, just wanted to keep it consistent with other components.

Reply:
I agree that this one is kind of obvious, but as it is reflected in the storybook, it is great to have a short description for every props :D